composer.json

```diff
@@ -23,41 +23,41 @@
 	},
 	"require": {
 		"php": "^8.1",
-		"typo3/cms-backend": "^12.4",
-		"typo3/cms-core": "^12.4"
+		"typo3/cms-backend": "^13.3.1 || ^13.4 || dev-main",
```

Comment:
IMO the declaration of `^13.4` is not necessary here because the `^13.3.1` should include the version 13.4. After the release of ^13.4 next week, this could be adjusted then.

Reply:
yeah, I thought it looks terrible, LOL.
Waiting for another opinion.